<launch>
    <arg name="platform" default="sim" />
    <!-- <arg name="bond_id_namespace" default="safety_bonds" /> -->

    <node pkg="iarc7_vision"
        type="floor_detector.py"
        name="floor_detector">
        <rosparam command="load" file="$(find iarc7_vision)/param/floor_detector_$(arg platform).yaml" />
    </node>
</launch>
